examples/open_loop_control_urdfbot/generate_actions.py

```diff
@@ -0,0 +1,87 @@
+#
+# Copyright(c) 2022 Intel. Licensed under the MIT License <http://opensource.org/licenses/MIT>.
+#
+
+# generate series of actions for fetch agent on different parts such as moving, arm and gripper.
+
+import argparse
+import os
+import numpy as np
+import pandas as pd
+
+# fetch arm poses come from https://github.com/StanfordVL/iGibson/blob/master/igibson/robots/fetch.py#L100
+arm_poses = {
+    "init": np.array([0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0]),
+    "default": np.array([-1.17079, 1.47079, 0.4, 1.67079, 0.0, 1.57079, 0.0]),
+    "vertical": np.array([-0.94121, -0.64134, 1.55186, 1.65672, -0.93218, 1.53416, 2.14474]),
+    "diagonal15": np.array([-0.95587, -0.34778, 1.46388, 1.47821, -0.93813, 1.4587, 1.9939]),
+    "diagonal30": np.array([-1.06595, -0.22184, 1.53448, 1.46076, -0.84995, 1.36904, 1.90996]),
+    "diagonal45": np.array([-1.11479, -0.0685, 1.5696, 1.37304, -0.74273, 1.3983, 1.79618]),
+    "horizontal": np.array([-1.43016, 0.20965, 1.86816, 1.77576, -0.27289, 1.31715, 2.01226]),
+}
+
+
+def add_actions(move_forward=0.0, move_right=0.0, gripper_state=50.0, arm_current="init", arm_next="init", arm_alpha=0.0):
+    action_map = {}
+
+    # base joints
+    action_map["joint.wheel_joint_r"] = move_forward - move_right
+    action_map["joint.wheel_joint_l"] = move_forward + move_right
+
+    # arm joints
+    arm_pose = arm_poses[arm_current] * (1.0 - arm_alpha) + arm_poses[arm_next] * arm_alpha
+    action_map["joint.arm_joint_0"] = arm_pose[0]
+    action_map["joint.arm_joint_1"] = arm_pose[1]
+    action_map["joint.arm_joint_2"] = arm_pose[2]
+    action_map["joint.arm_joint_3"] = arm_pose[3]
+    action_map["joint.arm_joint_4"] = arm_pose[4]
+    action_map["joint.arm_joint_5"] = arm_pose[5]
+    action_map["joint.arm_joint_6"] = arm_pose[6]
+
+    # gripper joints
+    action_map["joint.gripper_finger_joint_r"] = gripper_state
+    action_map["joint.gripper_finger_joint_l"] = gripper_state
+
+    # other joints
+    action_map["joint.head_pan_joint"] = 0.0
+    action_map["joint.head_tilt_joint"] = 0.0
+
+    return pd.DataFrame(action_map, index=[0])
```

Comment:
I don't think you need to include index=[0] because pandas will do that by default.

Reply:
That is the first thing I tried but `pd.DataFrame(action_map)` will cause an error for some : `ValueError: If using all scalar values, you must pass an index`.

Reply:
Another work around would be use list instead of float in `add_actions`.